### ISSUE TITLE
Tool to Test Homepage Layouts

### ISSUE THREAD

**Issue by vsinghkpcc** [SCPR/SCPRv4#328]
AC

1) Clone the KPCC Reader
2) Account for ad slots and requirements (A&B need to be "above the fold" and C should be a few slots down from A&B)
3) Represent basic listen live and site navigation
4) Programs need some type of basic representation
5) Article links should go directly to the scpr.org canonical url (this is a test of the homepage only)

Note: Ability to divert traffic for these tests, as well as event tracking, will be addressed in separate tickets.

**Comment by Ravenstine** [comment]
Putting this aside until we know where we're going with this.

**Comment by sdillingham** [comment]
Per our conversation in Slack, I'm scaling this ticket back in terms of our user testing plan. Rather than diverting a percentage of production traffic to the modified KPCC Reader concept, we will recruit a small group of homepage news consumers and test this specific concept with them. We will observe how they use our current homepage, then test an infinite scroll in a more directed user test. This allows us to learn something useful about what impact a change like this would have on users, but avoid some of the pitfalls. 

@Ravenstine: I think AC #4 (program representation) can be removed from this ticket. Also, we wouldn't create separate tickets for event tracking and diverting traffic since we'll be testing with a small number of users in a controlled test environment.

Other than that, I think the rest of the AC should stay the same: representing the global site nav is important so testers aren't too confused/disoriented, and the ad slots should be there so we don't get any false positives because users like a design that doesn't have ads.

**Comment by Ravenstine** [comment]
I'm really not finding any alternatives that do what we want that also don't have the same problems(such as pricing) as Optimizely.  @sdillingham and I have concluded that Google Content Experiments won't suffice, at least without jumping through multiple hoops.  At the same time, any open source A/B testing implementations would require as much or more work as if we used Google.  I'm going to put this ticket in the 'done' bucket, though if I come across anything new before the end of the sprint, I'll add it here.
